CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ Try to use the following format:
 ### Added
 
 ### Fixed
-
+- Fixed exporting reduced mt_bam to scout config
```

Comment:
We try to preserve these lines so please make a soup and insert the changes below